projects/core/src/cart/facade/active-cart.service.ts

```diff
@@ -498,6 +499,17 @@ export class ActiveCartService implements OnDestroy {
     });
   }
 
+  /**
+   * Triggers the refresh of the cart with new data from the backend
+   */
+  refreshCart(userId: string, cartId: string): void {
```

Comment:
There is `reloadCart` and `loadCart` in `multi-cart.service`, which does the same thing.

Reply:
Fixed.  The solution does not add a `refreshCart` function anymore, but rather the effects returns an additional action.